.gitignore

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
this line was added by gitignore.io, it would be better to not remove it

Reply:
yes